File: server/medicore.layer/MediCore.Service/Model/PolyClinicToHospitalMapModel.cs
using dna.core.service.Models.Abstract;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace MediCore.Service.Model
{
    public class PolyClinicToHospitalMapModel : IModelBase
    {
        public int Id { get; set; }

        [Required]
        public int HospitalId { get; set; }

        [Required]
        public int PolyClinicId { get; set; }
       
        public virtual PolyClinicModel PolyClinic { get; set; }
    }
}

File: smartmedika.admin/src/app/model/polyClinicToHospitalMapModel.ts
import { PolyClinicModel }  from './polyClinicModel';
import { IEntityBase }      from './IEntityBase';

export class PolyClinicToHospitalMapModel implements IEntityBase{
    id: number;
    hospitalId: number;
    polyClinicId: number;
    polyClinic: PolyClinicModel;

    constructor(id?:number, hospitalId?:number, polyClinicId?:number){
        this.id = id;
        this.hospitalId = hospitalId;
        this.polyClinicId = polyClinicId;
        
        this.polyClinic = null;
    }
}   